### PR TITLE
RC-195 Localize suggestion

### PR DESCRIPTION
**Pull request by GrimStal**

### CODE REVIEW

File: templates/page/defaultSuggestion.html [hunk 1]
```diff
@@ -0,0 +1,25 @@
+<!DOCTYPE html>
+<html lang="no">
```
Comment (phgroe) [inline templates/page/defaultSuggestion.html]: Can you change the `lang` attribute to get the value injected through the template locale parameter?

**Comment by phgroe** [comment]
Frickin Wercker is currently crashing in an early stage of the build, before even pulling the repository. Can't do nothing about that!

File: templates/page/defaultSuggestion.html (hunk 1 above)
Comment (GrimStal) [reply templates/page/defaultSuggestion.html]: Done

**Comment by GrimStal** [comment]
Merged in RC-182 PR

**Pull request closed without merging**